### ISSUE TITLE
Ratelimit question

### ISSUE THREAD

**Issue by setiseta** [vstakhov/rspamd#1792]
### Classification (Please choose *one* option):

* [ ] Crash/Hang/Data loss
* [ ] WebUI/Usability
* [ ] Serious bug
* [ ] Other bug
* [ ] Feature
* [ ] Enhancement
* [x] Question

### Reproducibility (Please choose *one* option):

* [X] Always
* [ ] Sometimes
* [ ] Rarely
* [ ] Unable
* [ ] I didn’t try
* [ ] Not applicable

### Rspamd version:
1.6.3

### Operation system, CPU, memory and environment:
Ubuntu 16.04 LTS

### Description (Please provide a descriptive summary of the issue):
whitelisted_rcpts not working as expected

### Steps to reproduce:
add an adress to whitelisted_rcpts

### Expected results:
no soft rejects if rcpt is this adress

### Actual results:
soft rejects cause of rate limits

### Additional information:
i have an adresse where bounces from newsletters go. i want to exclude this adress from rate limit cause there can come alot bounces. so i added it to the whitelisted_rcpts.
maybe this is the wrong place.

**Comment by fatalbanana** [comment]
That's a bug. [This commit](https://github.com/vstakhov/rspamd/commit/2d4a07cf3b36bd5e6c66e8d3847d387352f136a7) is relevant.

**Comment by setiseta** [comment]
Ok thank you for response.

**Comment by setiseta** [comment]
Seems still not working.

**Comment by setiseta** [comment]
I have configured following file:
/etc/rspamd/override/ratelimit.conf:
```
whitelisted_rcpts = "postmaster,mailer-daemon,ratelimitexcluded@example.com"
```
but the ratelimitexcluded@example.com still soft rejects.

**Comment by vstakhov** [comment]
It is not expected to work with user@domain, just user on its own.

**Comment by setiseta** [comment]
Ok, I've changed it to the userpart only.
But still the soft rejects.
I have restarted rspamd, so that the config gets reloaded instantly.
/etc/rspamd/override.d/ratelimit.conf:
```
whitelisted_rcpts = "postmaster,mailer-daemon,log.system";
```
Is it at a wrong place?

**Comment by moisseev** [comment]
I believe proper place is `/etc/rspamd/local.d/ratelimit.conf`.

`user@domain` matching is added in `master`: https://github.com/vstakhov/rspamd/pull/1787

**Comment by moisseev** [comment]
I'm afraid `whitelisted_rcpts` is broken in 1.6.
It is fixed in the `master`  https://github.com/vstakhov/rspamd/pull/1787/commits/2d4a07cf3b36bd5e6c66e8d3847d387352f136a7 but wasn't backported.

**Comment by setiseta** [comment]
Oh, ok, the fix is in the master, and not in the new release from yesterday?

**Comment by moisseev** [comment]
Yes, you are right.

**Comment by fatalbanana** [comment]
Should be fixed as of 1.6.5.
